script.service.next-episode/libs/commands.py

```diff
@@ -0,0 +1,232 @@
+# coding: utf-8
+# Created on: 17.03.2016
+# Author: Roman Miroshnychenko aka Roman V.M. (romanvm@yandex.ua)
+# License: GPL v. 3 <http://www.gnu.org/licenses/gpl-3.0.en.html>
+
+import os
+import sys
+from copy import deepcopy
+import xbmc
+from xbmcaddon import Addon
+from xbmcgui import Dialog
+import pyxbmct
+from medialibrary import (get_movies, get_tvshows, get_episodes, get_recent_movies,
+                          get_recent_episodes, get_tvdb_id, NoDataError)
+from nextepisode import (prepare_movies_list, prepare_episodes_list, update_data,
+                         get_password_hash, LoginError, DataUpdateError)
+from gui import NextEpDialog, ui_string
+
+addon = Addon('script.service.next-episode')
```

Comment:
Not quite true. Since this file is launched as a standalone script, addon ids must be provided explicitly.